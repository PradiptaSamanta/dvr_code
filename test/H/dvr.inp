dvr
rlimits  0.0 50.0
mapped-grid 0.005
n-grids 200
n-gl 5
rot-qn-max 3
num-eig-value-fac 0.5
nuclear-charge 1
mass 1.0
dvr-diag
dvr-integrals
direct-integrals
debug 6
enddvr

orbital
two-e-int 2
num-n-qn 4

endorbital

end
