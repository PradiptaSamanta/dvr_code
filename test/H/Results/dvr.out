 Reading the input from the file : dvr.inp
 **********
 Setting up these parameters:
 para%r_min     =     0.00
 para%r_max     =    20.00
 para%m         =       40
 para%nl        =        5
 para%nr        =      201
 para%l_max     =        3
 para%nev     =      101
 para%Z         =        1
 para%mass      =     1.00
 ***********
 Initializing the GLL grid ...
 Using potential from file input_pot.in
   
 Grid initialization is done.
 Solving the radial Schroedinger equation for l =            0
 Time taken for first step of diagonalization =    0.03600
 seconds.
 Time taken for second step of diagonalization =    0.02800
 seconds.
 Time taken for diagonalization =    0.06400
 seconds.
 Writing down the eigenvalues, eigenvectors and combining coefficients as demand
 ed ...
 Solving the radial Schroedinger equation for l =            1
 Time taken for first step of diagonalization =    0.03200
 seconds.
 Time taken for second step of diagonalization =    0.03200
 seconds.
 Time taken for diagonalization =    0.06400
 seconds.
 Writing down the eigenvalues, eigenvectors and combining coefficients as demand
 ed ...
 Solving the radial Schroedinger equation for l =            2
 Time taken for first step of diagonalization =    0.02800
 seconds.
 Time taken for second step of diagonalization =    0.02800
 seconds.
 Time taken for diagonalization =    0.05600
 seconds.
 Writing down the eigenvalues, eigenvectors and combining coefficients as demand
 ed ...
 Solving the radial Schroedinger equation for l =            3
 Time taken for first step of diagonalization =    0.02800
 seconds.
 Time taken for second step of diagonalization =    0.03200
 seconds.
 Time taken for diagonalization =    0.06000
 seconds.
 Writing down the eigenvalues, eigenvectors and combining coefficients as demand
 ed ...
 Calculating one-electron radial part of the integral for l =            0
 Calculating one-electron radial part of the integral for l =            1
 Calculating one-electron radial part of the integral for l =            2
 Calculating one-electron radial part of the integral for l =            3
 writing down the one-electron radial integrals
 Calculating two-electron radial part of the integral for l =            0
 Calculating two-electron radial part of the integral for l =            1
 Calculating two-electron radial part of the integral for l =            2
 Calculating two-electron radial part of the integral for l =            3
 writing down the two-electron radial integrals
 calculating the angular integrals
 Writing down the angular integrals
 **********
 Setting up these parameters for the orbitals:
 orb%n_max     =        8
 ***********
 Time taken for transformation =    0.00400
 seconds.
 Deallocated all the main matrices.
 The calculation is done. 
 Time =   0.888055000000000      seconds.
